contrib/babelfishpg_tsql/src/catalog.h

```diff
@@ -387,6 +387,20 @@ extern Oid	bbf_extended_properties_idx_oid;
 extern Oid	get_bbf_extended_properties_oid(void);
 extern Oid	get_bbf_extended_properties_idx_oid(void);
 
+typedef struct FormData_bbf_extended_properties
```

Comment:
reorder this struct definition as it was moved away from its function definition by previous commits